sentry-opentelemetry/sentry-opentelemetry-core/src/main/java/io/sentry/opentelemetry/PotelSentrySpanProcessor.java

```diff
@@ -0,0 +1,94 @@
+package io.sentry.opentelemetry;
+
+import static io.sentry.opentelemetry.InternalSemanticAttributes.IS_REMOTE_PARENT;
+import static io.sentry.opentelemetry.SentryOtelKeys.SENTRY_SCOPES_KEY;
+
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanContext;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.sdk.trace.ReadWriteSpan;
+import io.opentelemetry.sdk.trace.ReadableSpan;
+import io.opentelemetry.sdk.trace.SpanProcessor;
+import io.sentry.IScopes;
+import io.sentry.ScopesAdapter;
+import io.sentry.Sentry;
+import io.sentry.SentryLevel;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class PotelSentrySpanProcessor implements SpanProcessor {
```

Comment:
Indirectly related, but the old `SentrySpanProcessor` can be deleted as it isn't used anymore, right?

Reply:
We'll keep it around for a while and maybe deprecate it so users can upgrade to the new version when they need it. There should either be some config to switch between POTel and old way of OTel or separate dependencies.

Reply:
While the plan is now to replace the agent, I think we can simply deprecate and keep the old classes around for a bit.